packages/camera/camera_android/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Fixes unawaited_futures violations.
 
+## 0.10.9
```

Comment:
This should be 0.10.8+3 because the version < 1.0.0.

Reply:
@Mairramer Are you planning on changing this to `0.10.8+3 `? It also looks like you need to update the `pubspec.yaml` as well.

Reply:
I don't know if it's the most correct, but I put it as 0.10.9

Reply:
Since this is a bug fix for a plugin that has a version below `1.0.0`, we follow a policy of bumping the build version (e.g. `0.10.8+3`). If the version of the plugin was above `1.0.0` then bumping the patch version would be correct.

Reply:
i made the recommended change